pyproject.toml

```diff
@@ -16,6 +16,8 @@ langchain-core = ">=0.0.12"
 langchain-community = "^0.0.19"
 docugami = "^0.1.2"
 dgml-utils = "^0.3.2"
+pytest = "^7.3.0"
```

Comment:
I think this will make pytest and ruff required dependencies at runtime... but I dont think anyone who installs this package wants pytest and ruff. Can you make them optional dependencies only if somebody installs the test group? simple install of this package should not install these...